### ISSUE TITLE
Add "Last Seen" for the most malicious players

### ISSUE THREAD

**Issue by Thomas-Smyth** [Squad-Community-Ban-List/Squad-Community-Ban-List#23]
I've noticed that some of the most malicious players are not getting game banned by OWI despite reports being made, with evidence, to them about the player breaching basic rules, e.g. teamkilling, or breaking their Code of Conduct, e.g. racism. It would be good to find a way to better expose these players so we can pressure OWI to take more action.

I believe we could collect this data from the Battlemetrics API. However, their rate limit hugely affects our runtime which has a knock-on effect on the possibility of increasing the update frequency and our running costs. Thus, I think we will need to limit the number of players we track which is most obviously done by limiting it to the top x most harmful players or players with reputation points exceeding x.

**Comment by LeventHAN** [comment]
We can take the AVG of the threeshold servers use to ban players, that way we can be sure we track the correct toxic population...

**Comment by Thomas-Smyth** [comment]
I don't think we want to track that many people - maybe the top 50 just to start with.
